Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile produces a production ready image of talos-manager.
 
 ARG RUBY_VERSION=3.2.0
-FROM ruby:${RUBY_VERSION}-alpine as base
+FROM ruby:${RUBY_VERSION}-slim as base
```

Comment:
NOTE: Had to switch from `alpine` to `slim` to fix https://github.com/sparklemotion/sqlite3-ruby/issues/434